src/main/java/net/mcreator/element/converter/v2023_4/PlantGenerationConditionRemover.java

```diff
@@ -0,0 +1,190 @@
+	}
+
+	@Override public int getVersionConvertingTo() {
+		return 48;
+	}
+
+	// Returns the XML for the random patch block attached to the feature starting point
+	private static String getPatchXML(String plantType, int patchSize, String modElementName) {
+		if (plantType.equals("growapable")) {
+			return """
+					<block type="feature_random_patch" inline="false">
+						<field name="tries">%d</field>
+						<field name="xzSpread">7</field>
+						<field name="ySpread">3</field>
+						<value name="feature"><block type="placed_feature_inline">
+							<value name="feature"><block type="feature_block_column_simple">
+								<value name="height"><block type="int_provider_biased_to_bottom">
+									<field name="min">2</field><field name="max">4</field>
+								</block></value>
+								<value name="block"><block type="mcitem_allblocks"><field name="value">CUSTOM:%s</field></block></value>
+							</block></value>
+							<statement name="placement"><block type="placement_block_predicate_filter">
+								<value name="condition"><block type="block_predicate_all_of">
+									<mutation inputs="2"></mutation>
+									<value name="condition0"><block type="block_predicate_is_air"></block></value>
+									<value name="condition1"><block type="block_predicate_would_survive">
+										<field name="x">0</field><field name="y">0</field><field name="z">0</field>
+										<value name="block"><block type="mcitem_allblocks"><field name="value">CUSTOM:%s</field></block></value>
+									</block></value>
+								</block></value>
+							</block></statement>
+						</block></value>
+					</block>
+					""".formatted(patchSize, modElementName, modElementName);
+		} else {
+			return """
+					<block type="feature_random_patch_simple">
+						<field name="tries">%d</field><field name="xzSpread">7</field><field name="ySpread">3</field>
+						<value name="block"><block type="mcitem_allblocks"><field name="value">CUSTOM:%s</field></block></value>
+						<value name="condition"><block type="block_predicate_is_air"></block></value>
+					</block>
+					""".formatted(patchSize, modElementName);
+		}
+	}
+
+	// Returns the XML for the feature placement
+	private static String getPlacementXML(int frequency, boolean isRarer, boolean generateAtAnyHeight,
+			boolean onMotionBlocking) {
+		StringBuilder xml = new StringBuilder();
+		int blocksToClose = 2; // We always have the "In square" and one height placement blocks to close
+
+		// Add "Repeated x times" block if frequency on chunks isn't 1
+		if (frequency != 1) {
+			xml.append("<block type=\"placement_count\"><value name=\"count\"><block type=\"int_provider_constant\"><field name=\"value\">");
+			xml.append(frequency);
+			xml.append("</field></block></value><next>");
+			blocksToClose++;
+		}
+		// Add "With a chance of 1/32" block if plant is growapable or has flower-like generation
+		if (isRarer) {
+			xml.append("<block type=\"placement_rarity\"><field name=\"rarity\">32</field><next>");

```

Comment:
Some next statements are never closed, is this OK for the Blockly?

Reply:
The biome filter block doesn't have a next statement, since it's the last one

Reply:
That is ok. What I meant is that there is more `<next>` than `</next>` blocks in the generated XML

Reply:
Are there? `blocksToClose` starts at 2 to close the "Random XZ" and the height placement `<next>`, and the feature container block's `<next>` is closed at line 86

Reply:
Ok sorry yes you are correct, I missed you also increased the counter one line below my comment